### PR TITLE
Features/full/single listing sro updates #143897499

### PR DESCRIPTION
**Pull request by mluedke2**
Actually, don't review for a moment. I'm going to add tests...

### CODE REVIEW

**Comment by creativecoder** [comment]
Yes, our stated practice is that all "public" methods should have unit tests.

File: app/assets/javascripts/listings/ListingController.js.coffee [hunk 1]
```diff
@@ -206,8 +206,28 @@ ListingController = (
...
+  $scope.occupancyUnit = (maxOccupancy) ->
+    return 'person' if maxOccupancy == 1
+    'people'
+
+  $scope.formatBaths = (numberOfBathrooms) ->
+    return 'Shared' if numberOfBathrooms == 0
+    return '1/2 bath' if numberOfBathrooms == 0.5
+
+    fullBaths = Math.floor numberOfBathrooms
+    andAHalf = numberOfBathrooms - fullBaths == 0.5
```
Comment (creativecoder) [inline app/assets/javascripts/listings/ListingController.js.coffee]: @mluedke2 If you were feeling clever, you could do `andAHalf = numberOfBathrooms / .5 % 2`, but what you have is probably easier to read!

**Comment by mluedke2** [comment]
@creativecoder Ready for review now! I moved the "bath", "people", and "person" strings into the translation file, and added tests for all public methods

**Comment by mluedke2** [comment]
@creativecoder Sounds like a good idea. I just pushed that refactor.

**Comment by chrisdolendo** [comment]
:+1: thanks for the tests!

**Pull request merged**